### ISSUE TITLE
No validation Message when server image is not added

### ISSUE THREAD

**Issue by manavkhadka0** [AntonioErdeljac/next13-discord-clone#4]
# Validation Message During Server Creation

No validation message is seen during server creation if server image is not uploading.
Gives a buggy feeling if something is wrong with the api calls.

![image](https://github.com/AntonioErdeljac/next13-discord-clone/assets/61489414/d46c293c-458c-42c9-91e8-afc87847ed1f)

**Comment by sethumadh** [comment]
![code](https://github.com/AntonioErdeljac/next13-discord-clone/assets/63349857/dc366ee6-0bd7-4960-a332-6ad1dc5a1aec)
You just have to add  <FormMessage /> component  to the server image upload part.

**Comment by 8bitWatermelon** [comment]
To fix this issue, you must add a message to your image property in your form schema. it should look like something like this.

![image](https://github.com/AntonioErdeljac/next13-discord-clone/assets/107777766/dd86322f-2b09-4eff-a6f8-b19648e290b5)

Then in the form add a form message component provided by shadcn/ui.
![image](https://github.com/AntonioErdeljac/next13-discord-clone/assets/107777766/18658296-6b22-4969-a655-091452d0d9ed)

I hope this helps. 👋 Also please ignore the "Cable" I came up with the name and thank Antonio for the tutorial. 🙏

**Comment by 8bitWatermelon** [comment]
## The form message component is under the form control component.

**Comment by 8bitWatermelon** [comment]
Also another option is to use shadcn/ui toast to show a message if anything fails.
![image](https://github.com/AntonioErdeljac/next13-discord-clone/assets/107777766/bf7e80dd-e2ed-4f8e-a45b-f95c4bb74bac)

**Comment by manavkhadka0** [comment]
yes i created this issue to create a pr for it so that this will close after

**Comment by 1933874502** [comment]
@manavkhadka0 I followed your step，download docker desktop ect. But when I upload a server image，I got this error：Error: Invalid src prop [(https://utfs.io/f/02225339-0c16-46ce-afd8-2599ec503873_test.jpg)](http://localhost:3000/(https://utfs.io/f/02225339-0c16-46ce-afd8-2599ec503873_test.jpg)) on `next/image`, hostname "utfs.io" is not configured under images in your `next.config.js`

**Comment by Triv2** [comment]
> @manavkhadka0 I followed your step，download docker desktop ect. But when I upload a server image，I got this error：Error: Invalid src prop [(https://utfs.io/f/02225339-0c16-46ce-afd8-2599ec503873_test.jpg)](http://localhost:3000/(https://utfs.io/f/02225339-0c16-46ce-afd8-2599ec503873_test.jpg)) on `next/image`, hostname "utfs.io" is not configured under images in your `next.config.js`

Add `utfs.io` to your next.config.js like so:
![image](https://github.com/AntonioErdeljac/next13-discord-clone/assets/126743500/030e29aa-13bc-4bc5-92da-e2f8c580d939)
